backend/.env

```diff
@@ -0,0 +1,13 @@
+NEO4J_USER=neo4j
+NEO4J_PASSWORD=wordpass
+NEO4J_URI=bolt://localhost:7687
+
+JWT_SECRET=secret secret
+
+ADMIN_NAME=admin
+ADMIN_EMAIL=admin@aol.com
+ADMIN_PASSWORD=admin123
+
+TESTUSER_NAME=testuser
```

Comment:
Why do you parameterize that in a file? :thinking:

Just put that into the test case where it belongs.